frontend/src/views/FuelExports/FuelExportChangelog.jsx

```diff
@@ -66,6 +66,8 @@ export const FuelExportChangelog = () => {
   return (
     <Box>
       {changelogData?.map((item, i) => {
+        const isCurrentOrOriginalVersion =
+          i === 0 || item.nickname.includes('Original')
```

Comment:
I am a little hesitant to use nickname as an identifying variable. Can we not just use version = 0 instead? that would be more explicit.